modules/json/src/main/java/org/mule/module/json/transformers/TransformerInputs.java

```diff
@@ -101,6 +103,8 @@ public static TransformerFactory createTransformerFactory()
             transformerFactory = TransformerFactory.newInstance();
         }
 
+        XMLSecureFactories.createDefault().configureTransformerFactory(transformerFactory);
```

Comment:
If we don't configure both possible factories, the other implementation would be vulnerable.